_translator-files/po/fr/1-index.po

```diff
@@ -74,12 +74,12 @@ msgstr "Jamulus vous permet de jouer, de répéter ou de bœuffer avec vos amis,
 #. type: Attribute 'src' of: <div><div><figure><a><img>
 #: ../wiki/en/misc/1-index.md:32
 msgid "{% include img/en-screenshots/main-screen-medium.inc %}"
-msgstr ""
+msgstr "{% include img/en-screenshots/main-screen-medium.inc %}"
```

Comment:
We don't have a French screenshot for this?

Reply:
@trebmuh ?